Automation/update_flags.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
```

Comment:
The filename could be `update_countryflags` or `update_country_flags` to match the naming (`flags` alone sound like technical/feature flags)

Reply:
👍